examples/ngrok-tunnel/docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
         condition: service_started
     environment:
       - NGROK_TUNNEL_AUTO_DETECT=http://ngrok:4040
+      - PS_DOMAIN=localhost:8000
```

Comment:
Are you sure?
this shall be autodetected by the script.
https://github.com/PrestaShop/prestashop-flashlight/blob/main/assets/run.sh#L35C32-L35C56
https://github.com/PrestaShop/prestashop-flashlight/blob/main/assets/run.sh#L53

Reply:
Ok, it's maybe a problem with this part
https://github.com/PrestaShop/prestashop-flashlight/blob/main/assets/run.sh#L40-L42